doc/develop/test/ztest.rst

```diff
@@ -293,9 +293,9 @@ For example:
 Listing Tests
 =============
 
-Tests (test projects) in the Zephyr tree consist of many testcases that run as
+Tests (test projects) in the Zephyr tree consist of many test scenarios that run as
 part of a project and test similar functionality, for example an API or a
```

Comment:
This implies that 

- A "test" is "test project".
- A "test" contains many "test scenarios".
- "test scenarios" are part of a project.
- "test scenarios" test similar functionarlity.
- We have `test-identifier` which can be passed as a part of "full canonical name for each test scenario" with `-s` to `twister`.

The help doc for `-s` is:

```
  -s TEST, --test TEST, --scenario TEST
                        Run only the specified testsuite scenario. These are named by
                        <path/relative/to/Zephyr/base/section.name.in.testcase.yaml>
```